client/src/components/auth/Register.js

```diff
@@ -4,6 +4,7 @@ import { connect } from "react-redux";
 import { setAlert } from "../../actions/alert";
 import { register } from "../../actions/auth";
 import PropTypes from "prop-types";
+import {Form} from "react-bootstrap";
```

Comment:
I think it would be more consistent to follow the same coding pattern with the other lines of code, such as adding space before and after the `Form` => `import { Form } from "react-bootstrap";`